addons/mail/static/src/new/core/messaging_service.js

```diff
@@ -489,7 +489,7 @@ export class Messaging {
                 case "mail.message/toggle_star": {
                     const { message_ids: messageIds, starred } = notif.payload;
                     for (const messageId of messageIds) {
-                        const message = this.store.messages[messageId];
+                        const message = this.message.insert({ id: messageId });
                         if (!message) {
```

Comment:
this condition is always false after using insert

Reply:
You mean always `true` I guess :)